tikz/__init__.py

```diff
@@ -1117,6 +1121,7 @@ def _update(self):
         # document body
         codelines += [
             r'\begin{document}',
+            "\n".join(self.document_codes),
```

Comment:
It would be nicer to insert the elements of the list instead of the combined string. I believe `*self.document_code` should do the trick.